packages/web/pages/_app.tsx

```diff
@@ -39,6 +39,19 @@ const App: React.FC<WithUrqlProps> = ({
           />
         </>
       )}
+      {CONFIG.clarityID != null && (
```

Comment:
```suggestion
      {!!CONFIG.clarityID && (
```
would this be cleaner here ?

Reply:
Yeah, right.

Reply:
> Why not merge?

Rebase is the dev process we follow @manolingam  :)
With rebase only 1/2 commits of yours would be in this PR

Reply:
Just updated.